docker-compose/config/worker-ubuntu22-04.jsonnet

```diff
@@ -1,22 +1,80 @@
 local common = import 'common.libsonnet';
 
 {
-  blobstore: common.blobstore,
+  blobstore: {
+    actionCache: common.blobstore.actionCache,
+    contentAddressableStorage: {
+      readCaching: {
+        slow: common.blobstore.contentAddressableStorage,
+        fast: {
+            'local': {
+                keyLocationMapOnBlockDevice: {
+                file: {
+                    path: '/storage-worker-cas/key_location_map',
+                    sizeBytes: 100 * 1024 * 1024,
+                },
+                },
+                keyLocationMapMaximumGetAttempts: 8,
+                keyLocationMapMaximumPutAttempts: 32,
+                oldBlocks: 8,
+                currentBlocks: 24,
+                newBlocks: 1,
+                blocksOnBlockDevice: {
+                  source: {
+                      file: {
+                      path: '/storage-worker-cas/blocks',
+                      sizeBytes: 8 * 1024 * 1024 * 1024,
+                      },
+                  },
+                  spareBlocks: 3,
+                  dataIntegrityValidationCache: {
+                      cacheSize: 50000,
+                      cacheDuration: '14400s',
+                      cacheReplacementPolicy: 'LEAST_RECENTLY_USED',
+                  },
+                },
+                persistent: {
+                stateDirectoryPath: '/storage-worker-cas/persistent_state',
+                minimumEpochInterval: '300s',
+                },
+            },
+        },
+        replicator: { 'local' : {} },
```

Comment:
Could you use the following instead?
```
replicator: { deduplicating: { 'local': {} } },
```
This ensures that if multiple actions need to read the same file, those reads will be deduplicated against each other.